Gemfile

```diff
@@ -17,15 +17,14 @@ gem 'pg'
 gem 'jbuilder', '>= 2.2.13'
 
 # for authentication
-gem 'devise', '3.4.1'
-
-# for sending devise emails in background
-gem 'devise-async'
+gem 'devise' , '~> 4.1.0'
```

Comment:
Removed `devise-async` with reference to the issue https://github.com/bigbinary/aapc7atlis/pull/3531